pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -477,3 +481,18 @@ func waitLogs(ctx context.Context, id string, w *fsnotify.Watcher, runtimeServic
 		}
 	}
 }
+
+// retryOpen retries to open the file for 5 times with 1 second interval.
+func retryOpen(path string) (*os.File, error) {
+	var err error
+	for i := 0; i < 5; i++ {
+		var f *os.File
+		f, err = os.Open(path)
+		if err == nil {
+			return f, nil
+		}
+		klog.ErrorS(err, "Failed to open file, retrying", "path", path)
+		time.Sleep(1 * time.Second)
+	}
+	return nil, err
+}
```

Comment:
I'd say we wait with a backoff here, see:
https://github.com/kubernetes/kubernetes/blob/d220ecb415485980ff02b7a010031bf279e9e067/staging/src/k8s.io/apimachinery/pkg/util/wait/backoff.go#L446-L470

Reply:
Done.